auth/auth.go

```diff
@@ -179,10 +179,12 @@ func (c *Client) CustomTokenWithClaims(uid string, devClaims map[string]interfac
 
 // RevokeRefreshTokens revokes all refresh tokens for the specified user.
 //
-// RevokeRefreshTokens revokes all refresh tokens for the specified user.
-// In addition to revoking all refresh tokens for a user, all ID tokens issued
-// before revocation will also be revoked at the Auth backend. Any request with an
-// ID token generated before revocation will be rejected with a token expired error.
+// RevokeRefreshTokens updates the user's TokensValidAfterMillis to the current UTC second.
```

Comment:
Please document this from the developer's point of view, and not SDK's point of view. Something like:

```
 RevokeRefreshTokens revokes all refresh tokens issued to a user.

...
```

Reply:
This is pretty much in line with the Node docs. I am not sure I understand the comment. (i change the first line as suggested.)